zoo-hope/src/components/navBar/index.tsx

```diff
@@ -0,0 +1,72 @@
+import Nav from "react-bootstrap/Nav";
+import NavDropdown from "react-bootstrap/NavDropdown";
+import "../../styles/index.scss";
+import { useState } from "react";
+import LogoNavBar from "../../images/logo/logo.png"
+
+export const NavBar = () => {
+  const [clickedButtonUA, setClickedButtonUA] = useState(true);
+  const [clickedButtonEN, setClickedButtonEN] = useState(false);
+
+  const elements = [
```

Comment:
This data will be better to move out of file to new folder like src/data/
and create nav.ts
And add there that
So use just import from nav.ts to your file

Reply:
done